pkg/installer/uninstall.go

```diff
@@ -231,6 +248,32 @@ func (in *Installer) uninstallEtcdOperator() error {
 	return err
 }
 
+// checkForExistingWorkloads ensures no bound PVCs are using a storageos storageclass.
+func (in *Installer) checkForExistingWorkloads() error {
```

Comment:
I'd move this function into `utils/k8s` if you're going to use it for upgrades too.